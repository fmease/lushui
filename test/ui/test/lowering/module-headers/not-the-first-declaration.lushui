data Void: Type of

;; What is this?
module

