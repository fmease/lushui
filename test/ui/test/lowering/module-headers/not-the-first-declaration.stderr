error[E041]: the module header has to be the first declaration of the module
  ┌─ ${TEST_FOLDER}/lowering/module-headers/not-the-first-declaration.lushui:4:1
  │
4 │ module
  │ ══════
  │

error: aborting due to previous error

